flow/alerting/classifier.go

```diff
@@ -57,6 +57,9 @@ var (
 		// TODO(this is mostly done via NOTIFY_CONNECTIVITY, will remove later if not needed)
 		Class: "NOTIFY_CONNECT_TIMEOUT", action: NotifyUser,
 	}
+	ErrorNotifyGeneral = ErrorClass{
+		Class: "NOTIFY_GENERAL", action: NotifyTelemetry,
+	}
```

Comment:
Let's call it ErrorNormalize for now so that we can get more info during alerting
```suggestion
	ErrorNormalize = ErrorClass{
		Class: "NORMALIZE", action: NotifyTelemetry,
	}
```